tensorflow/lite/java/src/main/java/org/tensorflow/lite/NativeInterpreterWrapper.java

```diff
@@ -268,6 +272,11 @@ int getInputTensorCount() {
     return inputTensors.length;
   }
 
+  /** Gets the names of the input tensors. */
+  String[] getInputTensorNames() {
```

Comment:
Better to check if the name exists.

Reply:
Sure, I think the `init` method has to be called, but probably not a bad idea to add a check anyways.

Reply:
Should I just add this block before?
```
    if (inputTensorNames == null) {
      return new String[]
    }
```

Reply:
Either way looks good to me. inputTensorNames=new String[] {} seems better. What do you think?

Reply:
Done!

Reply:
@lu-wang-g done!